composer.json

```diff
@@ -24,6 +24,7 @@
 		"phpstan/phpstan": "self.version"
 	},
 	"require-dev": {
+		"ext-dom": "*",
```

Comment:
AFAIK this should not be added.

Reply:
What do you need it for in dev?

Reply:
@ondrejmirtes It's useful to do XML validation, see https://github.com/phpstan/phpstan-src/pull/65/files#diff-ba74c4a649ddce093d63acd494b761d6R157 but we can remove this assertion if you think is not useful